jobs/pending_payments.ts

```diff
@@ -134,6 +137,7 @@ exports.attemptCommunitiesPendingPayments = async (bot: Telegraf<MainContext>):
         request: pending.payment_request,
       });
       const user = await User.findById(pending.user_id);
+      if (user === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program